guides/common/attributes-satellite.adoc

```diff
@@ -112,6 +113,8 @@
 :SmartProxyServerTitle: {SmartProxyServer}
 :SmartProxyServers: {SmartProxyServer}s
 :SmartProxyServersTitle: {SmartProxyServers}
+:UpgradingDocTitle: Upgrading Connected {ProjectName} to {ProjectVersion}
+:UpgradingDisconnectedDocTitle: Upgrading Disconnected {ProjectName} to {ProjectVersion}
```

Comment:
I'm sorry, this was in the correct file `attributes-titles.adoc`, but it must go into the `satellite` section in that file.

Reply:
On second thought -- keep UpgradingDisconnectedDocTitle in the general section. That should fix the build error.

Reply:
@AkshayGadhaveRH ^^